mathesar/api/serializers/columns.py

```diff
@@ -95,7 +95,7 @@ class Meta(SimpleColumnSerializer.Meta):
         )
         model_fields = ('display_options',)
 
-    name = serializers.CharField(required=False)
+    name = serializers.CharField(required=False,allow_blank=True)
```

Comment:
Please add space after comma.
```suggestion
    name = serializers.CharField(required=False, allow_blank=True)
```